src/mbgl/gl/drawable_gl.cpp

```diff
@@ -240,27 +240,28 @@ gfx::StencilMode DrawableGL::makeStencilMode(PaintParameters& parameters) const
 }
 
 void DrawableGL::uploadTextures() const {
-    for (const auto& pair : textures) {
-        if (const auto& tex = pair.second) {
-            std::static_pointer_cast<gl::Texture2D>(tex)->upload();
+    for (const auto& texture : textures) {
+        if (texture) {
+            static_cast<gl::Texture2D&>(*texture).upload();
```

Comment:
Just curious, but why is the cast needed here?

Reply:
No needed, I just removed it. 👍

Reply:
OK it occurs in a few other places as well.